setup.py

```diff
@@ -54,26 +38,37 @@ def readfile(name):
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    python_requires=">=3.7",
+    python_requires=">=3.10",
     install_requires=[
         "plone.api",
         "Products.GenericSetup",
         "setuptools",
         "plone.restapi>=8.41.0",
         "plone.app.vocabularies>=4.3.0",
+        "plone.distribution",
+        "Products.CMFPlacefulWorkflow",
```

Comment:
Why does it depend on CMFPlacefulWorkflow?

Reply:
I do not remember anymore, but without this, package was breaking